setup.py

```diff
@@ -9,6 +9,12 @@
 with open('AUTHORS.rst') as authors_file:
     authors = authors_file.read()
 
+with open('requirements.txt') as f:
+    requirements = []
+    for library in f.read().splitlines():
+        requirements.append(library)
```

Comment:
I'd recommend not including `hypothesis` here (not needed to run your code). You can do something like this: https://github.com/Axelrod-Python/Axelrod/blob/master/setup.py#L7

(Also, even when it comes to loosening your dependency numbers I recommend hard pinning hypothesis.)